android/src/main/java/com/magicleap/magicscript/scene/nodes/views/ViewWrapper.kt

```diff
@@ -40,6 +42,14 @@ class ViewWrapper(context: Context, private val parent: UiNode) : LinearLayout(c
         layoutParams = LayoutParams(LayoutParams.WRAP_CONTENT, LayoutParams.WRAP_CONTENT)
     }
 
+    override fun onLayout(changed: Boolean, l: Int, t: Int, r: Int, b: Int) {
```

Comment:
Lets name the arguments:  `changed: Boolean, left: Int, top: Int, right: Int, bottom: Int`